packages/js/product-editor/src/components/variations-table/variations-table-row/variations-table-row.tsx

```diff
@@ -142,46 +141,44 @@ export function VariationsTableRow( {
 				className="woocommerce-product-variations__attributes-cell"
 				role="cell"
 			>
-				<MediaUpload
-					value={ variation.id }
-					onSelect={ ( image ) =>
-						handleChange(
-							[
-								{
-									id: variation.id,
-									image:
-										mapUploadImageToImage( image ) ||
-										undefined,
-								},
-							],
-							false
+				<ImageActionsMenu
+					selection={ [ variation ] }
+					onChange={ handleChange }
+					onDelete={ handleDelete }
+					renderToggle={ ( { onToggle, isBusy } ) =>
+						isBusy ? (
+							<div className="woocommerce-product-variations__add-image-button">
+								<Spinner
+									aria-label={ __(
+										'Loading image',
+										'woocommerce'
+									) }
+								/>
+							</div>
+						) : (
+							<Button
+								className={ classNames(
+									variation.image
+										? 'woocommerce-product-variations__image-button'
+										: 'woocommerce-product-variations__add-image-button'
+								) }
+								icon={ variation.image ? undefined : plus }
+								iconSize={ variation.image ? undefined : 16 }
+								// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+								// @ts-ignore this exists in the props but is not typed
+								size="compact"
+								onClick={ onToggle }
```

Comment:
I know it's not in the designs or in the ACs, but after testing this, I think that adding a label (or tooltip) mentioning what happens when pressing the `+` would be helpful for users.

It would look like this:

![Screenshot 2024-06-11 at 1 08 32 PM](https://github.com/woocommerce/woocommerce/assets/1314156/813ad519-cc90-487e-9624-9f2d9af91b17)

@j111q What do you think about that?

Reply:
Nice suggestion @octaedro. Nice to have an issue to cover just this!

Reply:
+1 Good addition!